std/stdio.d

```diff
@@ -2981,8 +2981,17 @@ $(D Range) that locks the file and allows fast writing to it.
     }
 
     /**
+     * Output range which locks the file when created, and unlocks it when it goes
+     * out of scope.
+     *
      * Returns: An $(REF_ALTTEXT output range, isOutputRange, std, range, primitives)
-     * that locks the file and allows fast writing to it.
+     * which accepts string types, `ubyte[]`, individual character types, and
+     * individual `ubyte`s.
+     *
+     * Note: Writing either arrays of `char`s or `ubyte`s is faster than
+     * writing each character individually from a range. For large files,
```

Comment:
Sentence talks about the speed increase writing to files resulting from writing in chunks. A minor clarification would be that this applies to writing to `stdout` as well as files.

Reply:
How about just "large amounts of data"?

Reply:
Sounds goods to me.